components/brave_wallet/browser/internal/orchard_block_scanner.cc

```diff
@@ -29,50 +33,57 @@ OrchardBlockScanner::~OrchardBlockScanner() = default;
 
 base::expected<OrchardBlockScanner::Result, OrchardBlockScanner::ErrorCode>
 OrchardBlockScanner::ScanBlocks(
-    std::vector<OrchardNote> known_notes,
+    OrchardTreeState tree_state,
     std::vector<zcash::mojom::CompactBlockPtr> blocks) {
+  std::unique_ptr<orchard::OrchardDecodedBlocksBundle> result =
+      decoder_->ScanBlocks(tree_state, blocks);
+  if (!result) {
+    DVLOG(1) << "Failed to parse block range.";
+    return base::unexpected(ErrorCode::kInputError);
+  }
+
+  if (!result->GetDiscoveredNotes()) {
+    DVLOG(1) << "Failed to resolve discovered notes.";
```

Comment:
If this message is important it should be part of the error, not a log. Debug logging should be removed prior to merge